features/admin.extensions.v1/configs/models/sms-provider.ts

```diff
@@ -16,11 +16,17 @@
  * under the License.
  */
 
+import { RequestErrorInterface } from "@wso2is/admin.core.v1/hooks/use-request";
+import { AxiosError } from "axios";
 import { ReactNode } from "react";
+import { SMSProviderAPIResponseInterface } from "../../../admin.sms-providers.v1/models/sms-providers";
```

Comment:
```suggestion
import { SMSProviderAPIResponseInterface } from "@wso2is/admin.sms-providers.v1/models/sms-providers";
```

Reply:
Tried using this but I get `Can't find module` error

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6405/commits/671f41432e3487233d78eddf4999fae8d33c82e8